k8_cortx_cloud/logs-cortx-cloud.sh

```diff
@@ -5,11 +5,38 @@ DIR=$(dirname "$SCRIPT")
 
 function parseSolution()
 {
-  echo "$($DIR/parse_scripts/parse_yaml.sh $DIR/$solution_yaml $1)"
+  echo "$($DIR/parse_scripts/parse_yaml.sh $solution_yaml $1)"
+}
+
+function usage() {
+  echo -e "\n** Script to Collect CORTX Cloud Logs for Support Bundle **\n"
+  echo -e "Usage: \`sh $0 [options]\`\n"
+  echo "Options:"
+  echo "    -s|--solution-config [FILE_PATH] : path of solution configuration file."
+  echo "                                       default file path is $solution_yaml."
+  echo "    -n|--nodename [NODENAME]: collects logs from pods running only on given node".
```

Comment:
where its mentioned that both these are args are optional

Reply:
Mentioned explicitly.